host/3.0/buster/amd64/java/java11/java11-slim.Dockerfile

```diff
@@ -49,9 +53,9 @@ ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
 
 COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
 COPY --from=runtime-image [ "/workers/java", "/azure-functions-host/workers/java" ]
-COPY --from=jre [ "/usr/lib/jvm/zre-hl-tools-11-azure-amd64", "/usr/lib/jvm/zre-11-azure-amd64" ]
+COPY --from=runtime-image [ "/usr/lib/jvm/jdk-11.0.12+7", "/usr/lib/jvm/jdk-11.0.12+7" ]
```

Comment:
We discussed this in our call. But, please find a better way to identify the location of the jdk to make this easier to update in the future. Something similar to the java version ENV could work. Or the solution we discussed of always placing the jdk in the same spot. 
Whatever you can do to limit the changes needed to update the java version would be great.

Reply:
Your updates should suffice here. thank you Kaibo